docs/howto/configure-authd.md

```diff
@@ -39,26 +39,80 @@ Finally, as the supported authentication mechanism is the device workflow, you n
 
 ## Broker configuration
 
-To configure the broker edit the file `/var/snap/authd-msentraid/current/broker.conf`: 
+To configure the broker edit the file `/var/snap/authd-msentraid/current/broker.conf`:
 
 ```ini
 [oidc]
 issuer = https://login.microsoftonline.com/<ISSUER_ID>/v2.0
 client_id = <CLIENT_ID>
 
 [users]
-# The directory where the home directory will be created for new users.
-# Existing users will keep their current directory.
-# The user home directory will be created in the format of {home_base_dir}/{username}
-# home_base_dir = /home
-
-# The username suffixes that are allowed to login via ssh without existing previously in the system.
-# The suffixes must be separated by commas.
-# ssh_allowed_suffixes = @example.com,@anotherexample.com
+## The directory where the home directory will be created for new users.
+## Existing users will keep their current directory.
+## The user home directory will be created in the format of {home_base_dir}/{username}
+#home_base_dir = /home
+
+## The username suffixes that are allowed to login via ssh without existing previously in the system.
+## The suffixes must be separated by commas.
+#ssh_allowed_suffixes = @example.com,@anotherexample.com
+
+## 'allowed_users' specifies the users who are permitted to log in after
+## successfully authenticating with the Identity Provider.
+## Values are separated by commas. Supported values:
+## - 'OWNER': Grants access to the user specified in the 'owner' option
+##        	(see below). This is the default.
+## - 'ALL': Grants access to all users who successfully authenticate
+##      	with the Identity Provider.
+## - <username>: Grants access to specific additional users
+##           	(e.g. user1@example.com).
+## Example: allowed_users = OWNER,user1@example.com,admin@example.com
+#allowed_users = OWNER
+
+## 'owner' specifies the user assigned the owner role. This user is
+## permitted to log in if 'OWNER' is included in the 'allowed_users'
+## option.
+##
+## If this option is left unset, the first user to successfully log in
+## via this broker will automatically be assigned the owner role. A
+## drop-in configuration file will be created in broker.conf.d/ to set
+## the 'owner' option.
+##
+## To disable automatic assignment, you can either:
+## 1. Explicitly set this option to an empty value (e.g. owner = "")
+## 2. Remove 'OWNER' from the 'allowed_users' option
+##
+## Example: owner = user2@example.com
+#owner =
```

Comment:
This is just a query from me:

Are `owner = ""` and `owner = ` both valid in the config file?

Reply:
Yes

Reply:
OK -- thanks for clarifying.